cmd/issueBrowse_test.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"testing"
+)
+
+func Test_issueBrowse(t *testing.T) {
```

Comment:
You'll need some additional setup here to read the config since its usually done in main
```
	viper.SetConfigName("lab")
	viper.SetConfigType("hcl")
	viper.AddConfigPath(".")
	err = viper.ReadInConfig()
```